airbyte-integrations/connectors/source-datadog/unit_tests/test_streams.py

```diff
@@ -52,6 +53,14 @@ def test_next_page_token(config):
     assert stream.next_page_token(**inputs) is None
 
 
+def test_site_config(config):
```

Comment:
@bartveenstra fwiw I was suggesting that we instantiate a stream which accepts `site`, call `url_base` and assert it is equal to what expect. This way we test that the value we pass into `site` controls the URL we end up using

Reply:
That would make more sense indeed. not that familiar with the code base yet nor saw a method that exposes the called url. I'll look into it